resume.html

```diff
@@ -1,67 +1,68 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <title>
-
-    <link rel="stylesheet" type="text/css" href="">
-</tittle>
+    <link rel="stylesheet" type="text/css" href="resume.css">
+    <title>Resume</title>
 </head>
 <body>
-	<div id="header">
-	<p id="name">Jyoti Sharma</p>
-         <a href="mailto:jyotiemailbox@gmail.com" target="_blank"><p id="email">jyotiemailbox@gmail.com</p></a>
+    <div id="header">
+	   <p id="name">Jyoti Sharma</p>
+        <a href="mailto:jyotiemailbox@gmail.com" target="_blank"><p id="email">jyotiemailbox@gmail.com</p></a>
+        <a href="contactto:8629844461" target="_blank"><p id="contact"><br><br>8629844461</p></a>
+        <a href="locationto:Mandi,India" target="_blank"><p id="location">Mandi, India</p></a>
     </div>
-    <div class="left">
-        <p id="Contact">8629844461</p>
-    </div>
-    <div class="right">
-        <h3>Carrier Objective</h3> 
-            <p>To solve problem in an effective manner</p>
-        <h3>CV Highlights</h3>
-            <p>
+    <div class ="right">
+    <h3>Carrier Objective</h3> 
+        <p>To solve problem in an effective manner</p>
+    <h3>CV Highlights</h3>
+        <p>
```

Comment:
what is the importance of `<p>` here? If you ignore this even then this will be all good.